doc/source/citations.rst

```diff
@@ -1,8 +1,8 @@
 Citations of documents: ``citations.yaml`` and ``cited-by.yaml`` files
 ----------------------------------------------------------------------
 
-Papis has support for downloading and exploring citations that documents reference,
-and also cited-by type references.
+Papis has support for downloading and exploring citations that documents
+reference, and also cited-by type references.
```

Comment:
Diffs like these explained: re-formatting to text-width of 80 cols, which looked to be kind of a standard in our docs.

Reply:
Yeah, ideally we would keep it at 88 cols like the code, but it's not enforced at the moment, so it's ok if some places are a bit off.